number/greater than tester/81881ae5-e545-4ff2-a9d4-ee8d86980a0f.json

```diff
@@ -1,20 +1,23 @@
 {
-  "description": "Checks if 'a' is greater than 'b'.",
+  "description": "Checks if the value received on `a` is greater than the value received on `b`. If the values are equal, it outputs false.\n\nExample:\n\n1. 20@0 is received via `a`\n2. 19@0 is received via `b`\n3. true@0 is sent via `greater`",
```

Comment:
Maybe rename the ports to `value` and `reference` to be more self describing and to be consistent with `number/greater than or equal tester`